src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,51 @@
+package roomescape.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.domain.Time;
+import roomescape.dto.TimeRequest;
+import roomescape.dto.TimeResponse;
+import roomescape.service.TimeService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class TimeController {
+    private final TimeService timeService;
+
+
+    @PostMapping("/times")
+    public ResponseEntity<TimeResponse> createTimes(@Valid @RequestBody TimeRequest request) {
+        Time newTime = timeService.registerTime(request.time());
+        return ResponseEntity.created(URI.create("/times/" + newTime.getId())).
+                body(TimeResponse.from(newTime));
+    }
+
+
+    @GetMapping("/times")
+    public ResponseEntity<List<TimeResponse>> readTimes() {
+        List<Time> times = timeService.getTime();
+
+        List<TimeResponse> responses = times.stream().
+                map(TimeResponse::from).toList();
+
+        return ResponseEntity
+                .ok()
+                .body(responses);
+    }
```

Comment:
```suggestion
    @GetMapping("/times")
    public List<TimeResponse> readTimes() {
        List<Time> times = timeService.getTime();

        List<TimeResponse> responses = times.stream().
                map(TimeResponse::from).toList();

        return responses;
    }
```

200 OK면서 단순 리턴하는 경우에는 수정된 코드랑 똑같은 결과가 나오는데요. 요거는 시간 나실 때 이유도 찾아보시면 좋아보입니다

Reply:
@RestController는 HttpMessageConverter를 통해 객체를 자동으로 JSON으로 직렬화해준다는 점 확인했습니다.
제가 작성한 ResponseEntity 방식은 상태 코드(200 OK)와 응답 body를 명시적으로 설정한 것이고,
리뷰어님께서 주신 방식은 객체를 그대로 반환하면 Spring이 직렬화를 자동으로 처리하는 형태라고 이해했습니다.
두 방식 모두 결과적으로 동일한 JSON 응답을 반환하는 구조라는 점도 확인했습니다!

Reply:
코드 가독성을 높이기 위해 제안해주신 방식대로 리팩토링해보았습니다!